python_modules/libraries/dagster-dbt/dagster_dbt/include/macros/dagster__log_columns_in_relation.sql

```diff
@@ -0,0 +1,11 @@
+{% macro dagster__log_columns_in_relation() %}
+    {%- set columns = adapter.get_columns_in_relation(this) -%}
+    {%- set table_schema = {} -%}
+
+    {% for column in columns %}
+        {%- set serializable_column = {column.name: {'data_type': column.data_type}} -%}
```

Comment:
Should/can we collect column descriptions here as well?

Another option would be to somehow merge them in later from definition metadata. I imagine that if they're specified at all, it will be in the dbt model config.

Reply:
[The API](https://docs.getdbt.com/reference/dbt-classes#column) does not have column descriptions, only name and data type.

Yeah, they'll be on the model config if specified. The data warehouse table information will only have description if the [user explicitly persists](https://docs.getdbt.com/reference/resource-configs/persist_docs) their dbt descriptions into their warehouse.